reservation/models.py

```diff
@@ -56,10 +56,11 @@ class Reservation(models.Model):
     playing_time = models.ForeignKey(PlayingTime, null=True, on_delete=models.CASCADE, verbose_name='Cinema/Hall/Date')
     is_confirmed = models.BooleanField(default=False)
     expired = models.BooleanField(default=False)
+    price = models.FloatField(default=20.0)
+    stripe_price_id = models.CharField(default='price_1LptmkGMvPFpBlyfXSbzeFXF', max_length=500)
```

Comment:
Not sure what the value represents: price_1LptmkGMvPFpBlyfXSbzeFXF. The default parameter also accepts a callable, if needed. The value could be as well added in the settings file.
So, what I would do is define a function that returns this value from the settings file and use it for the default value.
https://docs.djangoproject.com/en/dev/ref/models/fields/#default

Regarding the price, I didn't see a way this is calculated depending on the number of seats. Sorry if this is already done! You could customize this and make discounts if the number of seats is greater than N.

Reply:
price_1LptmkGMvPFpBlyfXSbzeFXF represents the product price id from stripe dashboard (you have to create the product there too).  You have to set this value in the post method when you create the session otherwise stripe would not know what to bill. You can check out the flow https://staging-cinema.herokuapp.com/reservations/landing/